en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/quick-start-guide/quick-start-guide-kubernetes-with-apim.md

```diff
@@ -119,7 +119,7 @@ Let's create our first project with the name "petstore" by adding the [OpenAPI d
 
     1. Access the Publisher in WSO2 API Manager.
     
-         [https://apim.wso2.com/publisher/apis](https://apim.wso2.com/publisher/apis)
+         [https://apim:9443/publisher/apis](https://apim:9443/publisher/apis)
```

Comment:
@pubudu538 , please confirm whether this address is correct. Just wondering whether some other URL is used when APIM is run on Kubernetes.